From b77d516b87740b147b8821cf151ee463a4f05d94 Mon Sep 17 00:00:00 2001
From: stakiar <stakiar@cloudera.com>
Date: Fri, 3 Mar 2017 17:55:21 -0800
Subject: [PATCH 1010/1164] CLOUDERA-BUILD: CDH-50662: Import/Export to ADLS
 failure

Adding "adl" to the list of supported schemes for export / import
commands in Hive. The upstream community is still working on
integration tests for export / import against blobstores, so
we are waiting on those tests to be completed before making
this change upstream.

Change-Id: Ica3e73a1f404180faa4520004b05d0273b38afd3
---
 .../java/org/apache/hadoop/hive/conf/HiveConf.java |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java b/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
index 6ff9a50..c06598a 100644
--- a/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
+++ b/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
@@ -1592,7 +1592,7 @@ public void setSparkConfigUpdated(boolean isSparkConfigUpdated) {
         "When true the HDFS location stored in the index file will be ignored at runtime.\n" +
         "If the data got moved or the name of the cluster got changed, the index data should still be usable."),
 
-    HIVE_EXIM_URI_SCHEME_WL("hive.exim.uri.scheme.whitelist", "hdfs,pfile,s3a",
+    HIVE_EXIM_URI_SCHEME_WL("hive.exim.uri.scheme.whitelist", "hdfs,pfile,s3a,adl",
         "A comma separated list of acceptable URI schemes for import and export."),
     // temporary variable for testing. This is added just to turn off this feature in case of a bug in
     // deployment. It has not been documented in hive-default.xml intentionally, this should be removed
-- 
1.7.9.5

